docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3.3"
+services:
+  postgres:
+    image: quay.io/azavea/postgis:2.2-postgres9.5-slim
+    environment:
+      - POSTGRES_DB=district_builder
+      - POSTGRES_USER=district_builder
+      - POSTGRES_PASSWORD=district_builder
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready"]
```

Comment:
We've seen instances of this failing when the service is up because of the missing `-U` argument to `pg_isready`.

Reply:
I can confirm that I'm seeing this after doing `docker-compose up`.

Reply:
I saw the error, but it didn't appear to affect anything -- I could still fire up a django shell and talk to the database so I thought of it as just an annoying thing. Fixing.

Reply:
This doesn't appear to be resolved by including `"-U", "district_builder" in the healthcheck command, though it does show up every 15 seconds, matching the healthcheck interval